tendrl/monitoring_integration/upgrades/delete_dashboards.py

```diff
@@ -52,15 +52,20 @@ def delete_dashboards(server_ip, user, password):
 
 def main():
     try:
+        print ("\n Modifying brick path separator \n")
+        # Modifying brick path separator | to :
+        os.system(
+            "find /var/lib/carbon/whisper/tendrl/ -depth  "
+            "-type d -name '*|*' -execdir bash -c "
+            "'mv \"$1\" \"${1//|/:}\"' bash {} ';'"
+        )
         print ("\n Migrating graphite data \n")
```

Comment:
It will replace brick path separator from | to :

Reply:
Delete _dashbaord name is working, as per upgrade document this we are executing this python script as tendrl-upgrade so I feel file name is wrong but the place is correct.

Reply:
Ok